packages/create-astro/src/templates.ts

```diff
@@ -1,22 +1,5 @@
 export const TEMPLATES = [
-	{
-		title: 'Just the basics (recommended)',
-		value: 'basics',
-	},
-	{
-		title: 'Blog',
-		value: 'blog',
-	},
-	{
-		title: 'Portfolio',
-		value: 'portfolio',
-	},
-	{
-		title: 'Documentation Site',
-		value: 'docs',
-	},
-	{
-		title: 'Empty project',
-		value: 'minimal',
-	},
+	{ value: 'basics', title: 'a few best practices (recommended)' },
+	{ value: 'blog', title: 'a personal website starter kit' },
+	{ value: 'minimal', title: 'an empty project' },
```

Comment:
I like the idea of these going in some sort of intuitive order, like `empty -> basics -> blog` or reversed. I don't think recommended needs to be at the top.

I guess I don't feel too strongly here, but wanted to call it out explicitly.

Reply:
That's fair. I personally think recommended should always be at the top because users might be manually skipping through the prompts using `Enter`, so the default choice should match our recommendation.